requirements.txt

```diff
@@ -4,8 +4,10 @@ librosa>=0.8.0
 numpy>=1.18.3
 pandas>=1.0.3
 pocketsphinx
+pre-commit
```

Comment:
Maybe we can move this + pytest to requirements_training.txt so we don't include them in the `pip` package requirements?